kalliope/core/ConfigurationManager/SettingEditor.py

```diff
@@ -11,6 +11,20 @@
 class SettingEditor(object):
     """This Static class provides methods/functions to update properties from the Settings"""
 
+    @staticmethod
+    def _check_name_in_list_settings_entry(name_to_check, list_settings_entry):
+        """
+        manage object models : STT, TRIGGERS, TTS, PLAYERS because they have "name" attributes
+        :param name_to_check: name to find in the list_settings_entry ~kalliope.core.Models.settings.SettingsEntry.SettingsEntry
+        :param list_settings_entry: the list of SettingsEntry to inspect
+        :return: True if the name_to_check corresponds to a name in the SettingsEntry list provided.
+        """
+        found = False
+        for settings_entry in list_settings_entry:
+            if settings_entry.name == name_to_check:
+                found = True
```

Comment:
you could add a break here or return True directly. Parsing the rest of the list seems not needed once you've found one.